pom.xml

```diff
@@ -142,6 +142,11 @@
           two versions are being updated on the IP BOM.-->
     <version.com.fasterxml.jackson>2.6.2</version.com.fasterxml.jackson>
     <version.commons-cli>1.3.1</version.commons-cli>
+
+    <!-- JaCoCo plugin configurations: -->
+    <jacoco.haltOnFailure>false</jacoco.haltOnFailure>
+    <jacoco.line.coveredratio.minimum>0.9</jacoco.line.coveredratio.minimum>
+    <version.jacoco.plugin>0.7.9</version.jacoco.plugin>
```

Comment:
You need to submit a PR to ```droolsjbpm-build-bootstrap``` to change the version of JaCoCo used. You also do not need to define "halt on failure" or "line covered ration min" here (as they're no different to those in ```droolsjbpm-build-bootstrap```.

See https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L190